pickup/lib/s3.js

```diff
@@ -87,16 +73,24 @@ export class S3Uploader {
     /**
      * @typedef {import('node:stream').Readable} Readable
      * @param {Readable} body
+     * @param {AbortSignal} signal
      */
-    return async function (body) {
-      return uploadAndVerify({
+    return async function (body, signal) {
+      // Handles s3 multipart uploading
+      // see: https://github.com/aws/aws-sdk-js-v3/blob/main/lib/lib-storage/README.md
+      const s3Upload = new Upload({
         client: s3,
-        validationBucket,
-        destinationBucket,
-        key,
-        body,
-        cid
+        params: {
+          Metadata: { structure: 'complete' },
```

Comment:
I think this is usually upper case first.

```suggestion
          Metadata: { structure: 'Complete' },
```

Reply:
oh no! that line hasn't changed! just moved! which means it's been bad `complete` all this time. lucky we don't use it for anything eh?

Reply:
maybe can fix when we migrate them if we want to